src/pg_to_evalscript/javascript_datacube/DataCube.js

```diff
@@ -85,6 +96,27 @@ class DataCube {
         })
     }
 
+    extendDimensionWithData(dimension, dataToAdd) {
```

Comment:
This is super minor, and I was trying to be more consistent with this myself - I think it would be clearer to use `dimension` when the parameter is actually the name of the dimension, and `axis` when it's the index (number) of the dimension, as in this case. ffti

Reply:
We should be as consistent as possible yes. Fixed